docs/visual-basic/programming-guide/concepts/linq/programming-with-nodes.md

```diff
@@ -162,7 +162,6 @@ Console.WriteLine(doc.Nodes().Count())
 ```xml  
 <?xml version="1.0" encoding="utf-8" standalone="yes"?>  
 <Root />  
```

Comment:
The "1" is the output from line 157, so it needs to stay. But we could change the snippet language to just "txt" instead of "xml".

Reply:
oh I see that only now, the count property. Thank you @gewarren